pkg/controller/migmigration/task.go

```diff
@@ -468,6 +478,27 @@ func (t *Task) Run() error {
 		} else {
 			t.Requeue = PollReQ
 		}
+	case EnsureCanceled:
+		t.Owner.Status.SetCondition(migapi.Condition{
+			Type:     EnsureCanceled,
```

Comment:
I think a `Canceling` condition would be clearer and more symmetrical with `Running`.